reference/commands/report.rst

```diff
@@ -0,0 +1,10 @@
+conan report
+============
+
+
+The ``conan report`` command contains subcommands that return information about packages and libraries.
+
+.. toctree::
+   :maxdepth: 1
+
+   conan report diff: Get the differences between two recipes, also comparing their sources. <report/diff>
```

Comment:
This is incomplete and does not follow how we document the reference for other commands, please check any other command to complete this.